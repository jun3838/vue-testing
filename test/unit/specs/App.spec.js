import {shallow} from 'vue-test-utils';
import {createRenderer} from 'vue-server-renderer';
import App from '@/App.vue';

describe('App', () => {
  test('App renders', () => {
    let app = shallow(App);
    expect(app.find('#app')).toBeTruthy();
  });

  test('Snapshot', () => {
    const renderer = createRenderer();
    const wrapper = shallow(App);

    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
});
