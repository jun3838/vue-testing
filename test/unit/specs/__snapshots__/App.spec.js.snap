// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot 1`] = `"<div id=\\"app\\" data-server-rendered=\\"true\\"><router-view></router-view></div>"`;
